manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go

```diff
@@ -111,46 +111,40 @@ func orchestratorObjectName(cr *miqv1alpha1.ManageIQ) string {
 	return cr.Spec.AppName + "-orchestrator"
 }
 
-func addMessagingEnv(cr *miqv1alpha1.ManageIQ, c *corev1.Container) {
+func addMessagingEnv(cr *miqv1alpha1.ManageIQ, c *corev1.Container, client client.Client) {
 	if !*cr.Spec.DeployMessagingService {
 		return
 	}
 
 	messagingEnv := []corev1.EnvVar{
 		corev1.EnvVar{
-			Name: "MESSAGING_HOSTNAME",
-			ValueFrom: &corev1.EnvVarSource{
-				SecretKeyRef: &corev1.SecretKeySelector{
-					LocalObjectReference: corev1.LocalObjectReference{Name: kafkaSecretName(cr)},
-					Key:                  "hostname",
-				},
-			},
+			Name:  "MESSAGING_HOSTNAME",
+			Value: cr.Spec.AppName + "-kafka-bootstrap",
```

Comment:
Why `-bootstrap`? Is that the service name that the kafka operator creates for us?

Reply:
Yes this is the naming convention of the svc created by strimzi